docker-compose.yaml

```diff
@@ -16,15 +16,18 @@ services:
           devices:
             - driver: nvidia
               count: all
-              capabilities: [gpu]
+              capabilities: [ gpu ]
   copilot_proxy:
-    image: moyix/copilot_proxy:latest
+    # For dockerhub version
+    # image: moyix/copilot_proxy:latest
+    # command: python3 -m flask run --host=0.0.0.0 --port=5000
     # For local build
-#    build:
-#      context: .
-#      dockerfile: copilot_proxy/Dockerfile
-    command: uvicorn app:app --host $API_HOST --port $API_PORT
+    build:
+      context: .
+      dockerfile: copilot_proxy/Dockerfile
+    command: uvicorn app:app --host ${API_HOST} --port ${API_PORT}
     env_file:
-      - .env
+      # You can modify this env file to configure your container copilot_proxy environment
+      - copilot_proxy.env
```

Comment:
Why use this when there is already `example.env`? Which should be copied over to `.env` on a users machine. 

That is a best practice for production systems, this breaks that pattern.

Reply:
Here it is like default.env instead of example.env. Providing a default configuration in software I don't think is a bad idea in production.

Using a file that doesn't exist so that the project doesn't work out-of-the-box is kind of bad. Also, there does exist a need for a default env file in `launch.sh`.

Reply:
The reason why I added `.env` and gave an `example.env` is that next time someone adds a feature, maybe it requires some sensitive information, it will become harder to separate default values and sensitive values. So yeah, right now its not needed but it's future proofing and adhering to best practices.

Reply:
It appears that both approaches have pros and cons. But for your scheme, how to solve the problem that `launch.sh` needs to assign a default value (environment variable)?

Reply:
Oh, I figured out a way, and if this PR turns out to be meaningful, I'll explain it in detail later.

Now let's turn our attention to other issues first.